src/app/tasks/modals/upload-submission-modal/upload-submission-modal.tpl.html

```diff
@@ -118,13 +118,8 @@ <h5>Rationale</h5>
               </div>
             </div><!--/ilo-description-->
             <div class="col-sm-12 col-md-5">
-              <task-ilo-alignment-rater
-                readonly="false"
-                unit="task.unit()"
-                show-zero-rating="true"
-                ng-model="alignments[ilo.id]">
-              </task-ilo-alignment-rater>
-            </div><!--/alignment-rater-->
+              <task-ilo-alignment-rater ng-show="true" [readonly]="false" [show-labels]="true" [unit]="task.unit()" [alignment]="alignments[ilo.id]"></task-ilo-alignment-rater>
```

Comment:
Why do we have `ng-show="true"`?

Reply:
I forget to delete it. 
It's for testing purpose.
Before I remove the original component, both components are active so I can compare the behaviour and layout.